doc/PlatformSpecifics.md

```diff
@@ -30,7 +30,7 @@ To generate this file, following the following steps:
 <EnableLLVM>True</EnableLLVM>
```

Comment:
I can't comment on the line directly, but you definitely should update the steps to generate the aot profile, especially the command

Reply:
Also, on one of my project, I've also added some additional comments to explain that the net version should be the same as the one targeted on the project

Reply:
Can you share a link of the changes you made